### ISSUE TITLE
Suggestion: Make renaming anchor window less easy to bring up.

### ISSUE THREAD

**Issue by Rodg88** [castcrafter/travel_anchors#36]
It might be nice for the naming window to not open every time you right click on an anchor, eg like with a wand. I'm currently playing with mods that allow greater reach, and it means I can right click from quite a way away, which is kinda of annoying if i want to teleport to a nearby anchor, eg a floor up, but it just brings up the rename window.

Maybe requiring an empty hand or sneaking to rename could work?

**Comment by Filip-Tomasko** [comment]
Yes, I also have had this issue quite a few times which resulted in some anchors having a string of `wwwwwwwww`'s at the end of the name :D
